packages/local_auth/local_auth/example/android/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.1'
+        classpath 'com.android.tools.build:gradle:7.3.1'
```

Comment:
What does the error look like without doing this? Presumably plugin clients will have to do this as well, and I'm wondering how good the error message currently is.

Reply:
I need to finish the solution but I pasted the error here: https://docs.google.com/document/d/14ckvV5chTj3c2HDDaw_bdmzUzXa9cHURm0RPXWk5eX8/edit#bookmark=id.5isj41bmhgan

It's not a clear error, unfortunately. Maybe we can try to catch this error and throw a new one, but of course, this wouldn't be the only error that signals a Gradle bump is needed.

Reply:
Oof, that's super opaque. How old does someone's gradle version have to be to see that if we bump `core` here? I'm wondering if we should hold off on landing this until we have a better error message for this at the `flutter` tool level, or if only very old projects would hit this.

Reply:
Looks like it has to at least be 7.0. It also looks like it is the `compileSdkVersion` bump that is causing this, which is necessary for the `core` bump (so I could bump `fragment` separately). I do want to point out, though, that the error is not fatal, so I guess it's more like a warning. Either way, we could hold off on landing the `core` bump at least for now, since most projects would probably hit this.

Reply:
Oh, looks like Dependabot wanted to bump fragment again already so I'll land that one: https://github.com/flutter/plugins/pull/6645